README.md

```diff
@@ -21,44 +21,12 @@ For this example we will use mysql however the procedure shouldn't be too differ
 
 # How it works
 
-1. create a new repo with a package.json with at least:
-    ```
-    {
-      "name" : "<name-of-your-app>",
-      "version" : "<version>",
-      "scripts" : {
-          "install" : "./build.sh",
-          "start" : "./run.sh"
-      },
-      "engines": {
-          "node" : "^10"
-      }
-    }
-    ```
-2. then create both scripts, the install one will build grafana and the start one will start the grafana-server binary
-3. in the install script put:
-    ```
-    #!/bin/bash
-    
-    go get github.com/grafana/grafana
-    cd $GOPATH/src/github.com/grafana/grafana
-    go run build.go setup
-    go run build.go build
-    
-    yarn install --pure-lockfile
-    yarn dev
-    ```
-4. this will build both the back-end and the front-end of Grafana.
-5. add ```GOPATH=/home/bas/<app_id>/go_home``` in your environment variables where <app_id> is the id of the app on clever-cloud
-6. in the start script put:
-    ```
-    #!/bin/bash
-
-    cd $GOPATH/src/github.com/grafana/grafana
-    $GOPATH/bin/grafana-server
-    ```
-7. finally add ```GF_SERVER_HTTP_PORT=8080``` in your app environment variables to ensure Grafana starts on the correct port and **make sure you have enabled a dedicated build instance**
-8. Grafana is now ready to start however at this point if you restart the app, all data will be lost as it's not persisted in a database
+1. Use this repository as a JS clever cloud application. It will build and run grafana using the root `package.json` file. 
+2. Check the node version use in `package.json` (to set the grafana expected one, for Grafana 7.3.5 it's node version 12: https://github.com/grafana/grafana/blob/v7.3.5/package.json#L314 ).
+3. This will build both the back-end and the front-end of Grafana.
+4. Add ```GOPATH=/home/bas/<app_id>/go_home``` in your environment variables where <app_id> is the id of the app on clever-cloud.
+5. Finally add ```GF_SERVER_HTTP_PORT=8080``` in your app environment variables to ensure Grafana starts on the correct port and **make sure you have enabled a dedicated build instance**.
```

Comment:
Can we explain why we have to use a dedicated build ?

Reply:
It's not, this PR is to repair with the build version (for go1.16). I will do later a PR using Grafana official binaries

Reply:
Fine :wink: